core/src/autogluon/core/models/ensemble/stacker_ensemble_model.py

```diff
@@ -54,11 +54,7 @@ def __init__(
             base_model_types_dict = {}
         self.base_model_names = base_model_names
         self.base_models_dict: Dict[str, AbstractModel] = base_models_dict  # String name -> Model objects
-        self.base_model_paths_dict = base_model_paths_dict
-
-        # FIXME: DO NOT DO THIS, FIX ASAP
-        self.base_model_paths_dict = {k: PathConverter.to_relative(v) for k, v in self.base_model_paths_dict.items()}
-
+        self.base_model_paths_dict = {key: os.path.relpath(val, self.path) for key, val in base_model_paths_dict.items()}
```

Comment:
Very nice. I think this may even fix all of the issues in stacker ensemble model.